dist/src/main/assemblies/bin.xml

```diff
@@ -58,6 +58,11 @@
             <outputDirectory>conf</outputDirectory>
             <fileMode>0644</fileMode>
         </fileSet>
+        <fileSet>
```

Comment:
These changes seem to be useless since you have renamed the configuration file to `jvm.properties`.

Reply:
fixed